static/app/views/explore/savedQueries/savedQueriesTable.tsx

```diff
@@ -202,32 +214,46 @@ export function SavedQueriesTable({
               />
             </SavedEntityTable.Cell>
             <SavedEntityTable.Cell>
-              <Avatar user={query.createdBy} tooltip={query.createdBy.name} hasTooltip />
+              {query.isPrebuilt ? (
+                <Tooltip title={'Sentry'}>
+                  <ActivityAvatar type="system" size={20} />
+                </Tooltip>
+              ) : (
+                <Avatar
+                  user={query.createdBy}
+                  tooltip={query.createdBy?.name}
+                  hasTooltip
+                />
+              )}
             </SavedEntityTable.Cell>
             <SavedEntityTable.Cell>
               <SavedEntityTable.CellTimeSince date={query.lastVisited} />
             </SavedEntityTable.Cell>
             <SavedEntityTable.Cell hasButton>
               <SavedEntityTable.CellActions
                 items={[
-                  {
-                    key: 'rename',
-                    label: t('Rename'),
-                    onAction: () => {
-                      trackAnalytics('trace_explorer.save_query_modal', {
-                        action: 'open',
-                        save_type: 'rename_query',
-                        ui_source: 'table',
-                        organization,
-                      });
-                      openSaveQueryModal({
-                        organization,
-                        saveQuery: getHandleUpdateFromSavedQuery(query),
-                        name: query.name,
-                        source: 'table',
-                      });
-                    },
-                  },
+                  ...(query.isPrebuilt
+                    ? []
+                    : [
+                        {
+                          key: 'rename',
+                          label: t('Rename'),
+                          onAction: () => {
+                            trackAnalytics('trace_explorer.save_query_modal', {
+                              action: 'open',
+                              save_type: 'rename_query',
+                              ui_source: 'table',
+                              organization,
+                            });
```

Comment:
I think it might make sense to track analytics inside the open save query model. But if it requires adding props specific to analytics that might not be as clean